test-adapter/src/main/scala/org/scalajs/sbttestadapter/ScalaJSTask.scala

```diff
@@ -45,24 +46,38 @@ final class ScalaJSTask private (
     slave.send("execute:" + jsonToString(data))
 
     // Prepare result handler
-    val logBuffer = mutable.Buffer.empty[LogElement[_]]
+    // TODO rip this out once we know it doesn't garble the logs.
+    val shouldBufferLog = {
+      val propName = "org.scalajs.sbttestadapter.bufferlog"
```

Comment:
This should actually be `org.scalajs.testadapter.bufferlog`, i.e., without the `sbt` in `sbttestadapter`. That's the actual package name of the test adapter. (The directories are wrongly `sbttestadapter`, see #3002 about that.)